src/mudlet.pro

```diff
@@ -1385,8 +1385,12 @@ macx {
 
 win32 {
     # set the Windows binary icon
-    RC_ICONS = icons/mudlet_main_512x512_6XS_icon.ico
-
+    contains(BUILD, "-ptb.+") {
```

Comment:
Does `contains(variablename, value)` work with regular expressions? The [QMake documentation for *Test Functions*](https://doc.qt.io/qt-5/qmake-test-function-reference.html#contains-variablename-value) does not make that clear...

Reply:
I just copied it from 5 lines further down...

Reply:
Oh, looking at https://docs.microsoft.com/en-us/windows/win32/uxguide/vis-icons (It does refer to 32Bit applications on the *new* Vista even though it is dated 2018!) suggests more work is needed - that `.ico` file has multiple images contain within it and a single icon is not enough - or in the right format... :frowning_face: